docs/troubleshooting.asciidoc

```diff
@@ -211,6 +211,21 @@ at the incorrect jar file.
 The correct jar file to be pointing at should be in the form of `elastic-apm-agent-<version>.jar` and
 further information about how to download this file can be found <<setup-javaagent, in the manual setup instructions.>>
 
+[float]
+[[trouble-shooting-communication]]
+==== Communication with APM Server ====
+
+`unable to find valid certification path to requested target` - server authentication fails. Check out
+<<ssl-server-authentication>>.
```

Comment:
It currently fails on this one. How does it work on internal links? Does it depend on order of the pages being built? How do we deal with that?

Reply:
OK, @felixbarny answered - the page needs to be included (directly or indirectly) from the `index.asciidoc`